project/ProjectPlugin.scala

```diff
@@ -234,24 +241,86 @@ object ProjectPlugin extends AutoPlugin {
   import autoImport._
 
   override def projectSettings: Seq[Def.Setting[_]] =
-    // format: OFF
     sharedReleaseProcess ++ warnUnusedImport ++ Seq(
-      addCompilerPlugin(%%("paradise", V.paradise) cross CrossVersion.full),
-      libraryDependencies ++= commonDeps :+ %("slf4j-nop") % Test,
-      scalaVersion := "2.12.7",
-      crossScalaVersions := Seq("2.11.12", "2.12.7"),
+      description := "Freestyle RPC is a purely functional library for " +
```

Comment:
When are we changing the name?

Reply:
This is the very beginning of the process to change the name, the group id, etc.

Next stage: https://github.com/higherkindness/freestyle-rpc/pull/415 (still a WIP though).